R/db-sql.R

```diff
@@ -281,6 +281,29 @@ sql_query_join.DBIConnection <- function(con, x, y, vars, type = "inner", by = N
     con = con
   )
 }
+#' @export
+sql_query_join.SQLiteConnection <- function(con, x, y, vars, type = "inner", by = NULL, na_matches = FALSE, ...) {
+  if (type == "full") {
+    # workaround as SQLite doesn't support FULL JOIN
+    build_sql(
+      sql_query_join(con, x, y, vars, type = "left", by = by, na_matches = na_matches, ...),
```

Comment:
Are you sure this can go anywhere a join usually can?

Reply:
I wrapped the join in a subquery now so that should be safe to use in any place where a join can be.